forum/README.md

```diff
@@ -0,0 +1,20 @@
+# Using the Forum
+
+The [modding forum]() will be extremely helpful to learning how to mod and getting mods from others. You will need a [vivaldi account](https://login.vivaldi.net) to post there. You may also want to look at these git repositories for more mods:
+<TODO>
+
+Contents
```

Comment:
Please use the main README for contents (only refer to files, not their sections). If you feel like there is a folder README needed, keep it here as well though. Of course if you want it to keep track of your work, you can (until the PR is ready).

Reply:
Yes, until PR is ready, I want each section to be approved by someone else.

Reply:
I do like a readme in every major folder, but I will take the table of contents and all out